src/main/java/org/gridsuite/shortcircuit/server/ParametersController.java

```diff
@@ -0,0 +1,73 @@
+package org.gridsuite.shortcircuit.server;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.gridsuite.shortcircuit.server.dto.ShortCircuitParametersInfos;
+import org.gridsuite.shortcircuit.server.service.ShortCircuitService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.http.MediaType.TEXT_PLAIN_VALUE;
+
+@RestController
+@RequestMapping(path = "/" + ShortCircuitApi.API_VERSION + "/parameters")
+@Tag(name = "Short circuit server analysis parameters")
+public class ParametersController {
+    private final ShortCircuitService shortCircuitService;
+
+    public ParametersController(ShortCircuitService shortCircuitService) {
+        this.shortCircuitService = shortCircuitService;
+    }
+
+    @GetMapping(path = "/{parametersUuid}", produces = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Get the parameters for an analysis")
+    @ApiResponse(responseCode = "200", description = "The parameters asked")
+    @ApiResponse(responseCode = "404", description = "The parameters don't exists")
+    public ResponseEntity<ShortCircuitParametersInfos> getParameters(@Parameter(description = "UUID of parameters") @PathVariable("parametersUuid") UUID parametersUuid) {
+        return ResponseEntity.of(shortCircuitService.getParameters(parametersUuid));
+    }
+
+    @PostMapping(produces = TEXT_PLAIN_VALUE, consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "Create a new set of parameters for an analysis using default parameters if none given")
+    @ApiResponse(responseCode = "200", description = "The new parameters entity ID")
+    public ResponseEntity<UUID> createParameters(@Parameter(description = "Parameters to save instead of default ones", schema = @Schema(implementation = ShortCircuitParametersInfos.class))
+                                                 @RequestBody(required = false) ShortCircuitParametersInfos parameters) {
+        return ResponseEntity.ok(shortCircuitService.createParameters(parameters));
+    }
```

Comment:
Can you keep the same logic as other computation services with a dedicated endpoint for defaullt parameters creation ?